staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr_test.go

```diff
@@ -110,6 +110,61 @@ func TestIntOrStringMarshalJSONUnmarshalYAML(t *testing.T) {
 	}
 }
 
+func TestGetIntFromString(t *testing.T) {
+	tests := []struct {
+		input      IntOrString
+		expectErr  bool
+		expectVal  int
+		expectPerc bool
+	}{
+		{
+			input:      FromInt(200),
+			expectErr:  false,
+			expectVal:  200,
+			expectPerc: false,
+		},
+		{
+			input:      FromString("30%0"),
+			expectErr:  true,
+			expectPerc: false,
+		},
+		{
+			input:      FromString("40%"),
```

Comment:
Would be good to include a test case for `FromString("200")` as well to prove `expectPerc` would be false

Reply:
Commited the new tests, do they look good?
@tanjunchen